src/app/providers/authentication/authentication.constants.ts

```diff
@@ -1,6 +1,7 @@
 export enum AuthenticationError {
   NO_INTERNET = 'The Internet connection appears to be offline.',
-  USER_CANCELLED = 'User cancelled authentication flow',
+  // eslint-disable-next-line max-len
```

Comment:
Don't think we should get into the habit of disabling `eslint` for every little thing. It's gonna grow on us and we'll realise that we started to disable `eslint` for important things as well.

Divide the string in 2 and keep the line length under `120`